+srv/expServer.m

```diff
@@ -357,50 +344,81 @@ function calibrateWaterDelivery()
   end
 
   function whiteScreen()
+    % WHITESCREEN Change screen background to white
     rig.stimWindow.BackgroundColour = rig.stimWindow.White;
     rig.stimWindow.flip();
     rig.stimWindow.BackgroundColour = bgColour;
   end
 
   function calibrateGamma()
+    % CALIBRATEGAMMA Perform gamma correction and save to hardware file
+    %   Performs a gamma correction, applies the new values and saves them
+    %   to the rig hardware config file.
+    %
+    % See also saveGamma, hw.ptb.Window/calibration
+
     stimWindow = rig.stimWindow;
+    
+    % Parameters for calibration
```

Comment:
can you add comments for all the following variable declarations that serve as calibration parameters?

Reply:
The log should make these clear enough, otherwise it should be changed.  What would you propose it say?

Reply:
something like 'port to connect photodiode', 'first port to connect hook-up wire for chrono' 'second port to connect hook-up wire for chrono', 'synonomous port for `clockOut`'